data/prevalentai.mdx

```diff
@@ -0,0 +1,22 @@
+---
+name: Prevalent AI
+description: A space to solve problems, crack complexity and pioneer innovative solutions .
+founded: 2017
+based: Service
+domain: Data and Cyber Security
+logo: prevalent.jpeg
+location: Kochi
+address: Transasia Cyberpark
+  Infopark Phase2, Kochi,
+  Kerala IN
+technologies:
```

Comment:
✨Thank you @AlexyPulivelil for this. There is a small correction I want to point out. The technologies here refer to the software/library/language/framework that the company uses rather than the area or software domain that the company works on.

For example: 
- Pytorch, Tensorflow, Python for AI and ML
- Hadoop for Big Data

The technologies are subjective to the company. The rest of the content in the PR is perfect ❤️